### PR TITLE
Map trial spawner

### PR DESCRIPTION
**Pull request by MattiDragon**
I've begun mapping the trial spawner code. I don't have time to finish today, but I'd like some reviews already.

### CODE REVIEW

File: mappings/net/minecraft/client/render/block/entity/MobSpawnerBlockEntityRenderer.mapping [hunk 1]
```diff
@@ -2,3 +2,12 @@ CLASS net/minecraft/class_839 net/minecraft/client/render/block/entity/MobSpawne
 	FIELD field_38885 entityRenderDispatcher Lnet/minecraft/class_898;
 	METHOD <init> (Lnet/minecraft/class_5614$class_5615;)V
 		ARG 1 ctx
+	METHOD method_55253 render (FLnet/minecraft/class_4587;Lnet/minecraft/class_4597;ILnet/minecraft/class_1297;Lnet/minecraft/class_898;DD)V
+		ARG 0 tickDelta
+		ARG 1 matirces
```
Comment (Benonardo) [inline mappings/net/minecraft/client/render/block/entity/MobSpawnerBlockEntityRenderer.mapping]: Typo, `matirces` -> `matrices`

**Comment by Shnupbups** [comment]
Please remove the underscore from the package name, and put TrialSpawnerState in the block enum package

**Comment by MattiDragon** [comment]
Some notes: 
- There's an unmapped record class that looks like it holds trial spawner config. 
- There are still many unmapped methods

**Review by Shnupbups — APPROVED** [review]
Will resolve conflicts with my PR soon

**Pull request merged**